client/src/app/shared/video/video-miniature.component.scss

```diff
@@ -109,6 +106,34 @@ $more-margin-right: 15px;
     opacity: 1;
   }
 
+  &.fit-width {
+    width: 100%;
+    height: unset;
+    margin-bottom: $video-miniature-margin-bottom / 2;
+
+    .video-bottom {
+      width: 100% !important;
+
+      .video-miniature-information {
+        width: calc(100% - 40px) !important;
+      }
+    }
+
+    my-video-thumbnail {
```

Comment:
Could you explain this change? I'm not sure to understand why you need absolute position

Reply:
Yes, it is the same change I used on `.screenratio`, it provides 16:9 ratio without relying on width *and* height values being preset.